### DISCUSSION TITLE
In-app purchase not syncing among my devices

### DISCUSSION THREAD

**Discussion by carloronconi** [MobiusSync/MobiusSync#126] [General]
I've recently purchased the in-app upgrade on my iPhone. Today I installed the standard (non-pro) app on my iPad and thought the upgrade would automatically be available to my iPad as well (the iPad has the same iCloud account as the iPhone), but I'm being prompted to purchase the pro upgrade again.

**Comment by SimonPickup** [comment]
You need to choose Restore Purchases. Apple will not allow multiple purchases anyway.
